### PR TITLE
Added a post about lighting talks 17-10-2016

### PR DESCRIPTION
**Pull request by AyhamAlzoubi**

### CODE REVIEW

**Comment by AyhamAlzoubi** [comment]
@odino @joejean @hossam-fares @Mohamed-amin @oba11 @geshan can you guys review?

File: source/_posts/2016-10-17-lighting-talks-17-10-2016.markdown [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+[Disu](team/#Adedamola%20Disu) Showed us the benefits of The Great Suspender Chrome extension 
+
+[Filo](team/#filippo%20de%20santis) Mentioned Yarn package manager and Nock as mocking library
+
+And here are the slides:
+
+<iframe src="https://docs.google.com/presentation/d/1733hpKGtrz46rJPE44SObqsZWsWIoVsjDGBh9RlPsPc/embed?start=true&loop=false&delayms=3000" frameborder="0" width="960" height="569" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
+https://docs.google.com/presentation/d/1733hpKGtrz46rJPE44SObqsZWsWIoVsjDGBh9RlPsPc/pub?start=false&loop=false&delayms=3000
+
+<hr />
```
Comment (geshan) [inline source/_posts/2016-10-17-lighting-talks-17-10-2016.markdown]: is this line break needed?

File: source/_posts/2016-10-17-lighting-talks-17-10-2016.markdown [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Lightning talks #3"
+date: 2016-10-17
+author: Ayham Alzoubi
+comments: true
+categories: [Lightning talks, nodejs, promise, bash, generator, sync, await, breadth algorithm, dev tools, codecept, aiml, nock, yarn]
+---
+
+>At Namshi, as you probably know, we do several activities in parallel, like - small presentations about new technologies,
```
Comment (geshan) [inline source/_posts/2016-10-17-lighting-talks-17-10-2016.markdown]: I think we can skip `in parallel` @razanbilwani wdyt?

File: source/_posts/2016-10-17-lighting-talks-17-10-2016.markdown [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+>showcases of the latest projects, watching interesting conference talks, etc.
+>Today, we had our third edition in our traditional lighting talk. Our team members gave a small presentation about something they found interesting.
```
Comment (geshan) [inline source/_posts/2016-10-17-lighting-talks-17-10-2016.markdown]: `we had our third edition of lighting talk`

**Pull request merged**